CarLot/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+namespace CarLot
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            CarLot samCarlot = new CarLot("Sam's Carlot");
+            CarLot lameCarlot = new CarLot("Lame Carlot");
+            Truck truck1 = new Truck("B8D3F20", "Ford", "150", 15000, 7);
+            Truck truck2 = new Truck("K39C0W", "Chevy", "Silverado", 16000, 6);
+            Car car1 = new Car("Sedan", 4, "67BN21", "Nissan", "Altima", 14500);
+            Car car2 = new Car("Hatchback", 2, "KO92LO", "Honda", "Civic", 19450);
+            samCarlot.AddVehicle(truck1);
+            samCarlot.AddVehicle(car1);
+            lameCarlot.AddVehicle(truck2);
+            lameCarlot.AddVehicle(car2);
+            samCarlot.PrintInventory();
+            lameCarlot.PrintInventory();
+        }
+    }
+    public class CarLot
+    {
+        //name prop
+        string Name { get; set; }
+        //list <vehicle>
+        List<Vehicle> Vehicles = new List<Vehicle>();
+
+        public CarLot(string name)
+        {
+            this.Name = name;
+        }
+        public void AddVehicle(Vehicle vehicle)
+        {
+            Vehicles.Add(vehicle);
+        }
+        public void PrintInventory()
+        {
+            System.Console.WriteLine($"There are {Vehicles.Count} vehicles");
+            foreach (var vehicle in Vehicles)
+            {
+                vehicle.PrintDescription();
+            }
+
+        }
+    }
+    public abstract class Vehicle
+    {
+        public abstract string LicenseNumber { get; set; }
+        public abstract string Make { get; set; }
+        public abstract string Model { get; set; }
+        public abstract double Price { get; set; }
+
+        public abstract void PrintDescription();
+    }
+    public class Truck : Vehicle
+    {
+        //5 properties
+        int BedSize { get; set; }
+        public override string LicenseNumber { get; set; }
```

Comment:
when you don't declare the Vehicle.LicenseNumber as abstract then you don't need to override it here.